pkg/services/libraryelements/database.go

```diff
@@ -434,12 +444,28 @@ func (l *LibraryElementService) patchLibraryElement(c *models.ReqContext, cmd pa
 		if elementInDB.Version != cmd.Version {
 			return errLibraryElementVersionMismatch
 		}
+		updateUID := cmd.UID
+		if len(updateUID) == 0 {
+			updateUID = uid // if UID isn't provided in the PATCH request let's use the existing UID
+		}
+		if !util.IsValidShortUID(updateUID) {
```

Comment:
i just got a small concern here, since if someone patch update the uid with invalid ones, they would be blocked there, not able to correct, probably has to connect into DB and change the table. but that should be really minor cases for a new feature like panel library. Otherwise the PR LGTM.

Reply:
Changed the code so we don't check existing UIDs.